configsuite/schema.py

```diff
@@ -72,11 +78,19 @@ def assert_valid_schema(schema):
         raise TypeError("Unknown base container: {}".format(schema))
 
 
-def _assert_valid_schema_level(schema):
+def _assert_valid_schema_level(schema, allow_default=True):
```

Comment:
This should not be an optional argument?

Reply:
fixed